src/components/Autocomplete.vue

```diff
@@ -7,6 +7,7 @@
             <button
               class="flex h-7 w-full items-center justify-between gap-2 rounded bg-gray-100 px-2 py-1 transition-colors hover:bg-gray-200 focus:ring-2 focus:ring-gray-400"
               :class="{ 'bg-gray-200': isComboboxOpen }"
```

Comment:
There might also be a conditional class application here to indicate that the line is disabled, such as replacing the current line with:
```
:class="[ {'bg-gray-200': isComboboxOpen}, disabled ? 'text-gray-600 bg-gray-50 placeholder-gray-400 hover:bg-gray-50' : '' ]"
```